src/constants.py

```diff
@@ -8,10 +8,13 @@
 from web3 import Web3
```

Comment:
What's the package for `gnosis` in `requirements.txt`?

Reply:
Ah, I see it's installed as a dependency from another package. Do we have to define it in the `requirements.txt` then?

Reply:
You may have highlighted the wrong line here because this is a web3 import.

The gnosis library that your IDE is insisting you install is actually an unknown library (that is not supported by gnosis and is suspected to be spam). The gnosis package is installed and included with `safe-eth-py`. 

I asked the same question to @Uxio0 when putting this together.